hpx/components/containers/partitioned_vector/detail/make_index_sequence.hpp

```diff
@@ -0,0 +1,113 @@
+//  Copyright (c) 2017 Antoine Tran Tan
```

Comment:
Documentation is in progress. But I need some documentation for `partitioned_vector` first to then make some increment concerning the views. The API is fully exposed in the following unit tests.

- https://github.com/atrantan/hpx/blob/partitioned_vector_view/tests/unit/component/partitioned_vector_view.cpp
- https://github.com/atrantan/hpx/blob/partitioned_vector_view/tests/unit/component/partitioned_vector_subview.cpp
- https://github.com/atrantan/hpx/blob/partitioned_vector_view/tests/unit/component/partitioned_vector_view_iterator.cpp